pkg/storage/stores/shipper/downloads/metrics.go

```diff
@@ -23,6 +26,17 @@ func newMetrics(r prometheus.Registerer) *metrics {
 			Name:      "query_time_table_download_duration_seconds",
 			Help:      "Time (in seconds) spent in downloading of files per table at query time",
 		}, []string{"table"}),
+		ensureQueryReadinessDurationSeconds: promauto.With(r).NewHistogram(prometheus.HistogramOpts{
+			Namespace: "loki_boltdb_shipper",
+			Name:      "query_readiness_duration_seconds",
+			Help:      "Time (in seconds) spent making an index gateway ready to be queried",
+			Buckets:   instrument.DefBuckets,
+		}),
+		usersToBeQueryReadyForTotal: promauto.With(r).NewGauge(prometheus.GaugeOpts{
+			Namespace: "loki_boltdb_shipper",
+			Name:      "users_to_be_query_ready_for_total",
+			Help:      "Total number of users an index gateway instance has to be query ready for.",
+		}),
```

Comment:
it could be confusing for some people that the metrics are namespaced with `loki_boltdb_shipper` but the help text mentions `index gateway`

Reply:
removed this one and adjusted the other helper text.